src/views/components/input/checkbox-2/InputCheckbox.js

```diff
@@ -7,32 +7,35 @@ class InputCheckbox extends Component {
   static defaultProps = {
```

Comment:
can this directory name be changed to `checkbox`?

Reply:
or removed entirely if it's superfluous

Reply:
getting this error when running. can you resolve this and the above issues then I can test+merge

```ERROR in ./config/routes/routes.web.js
Module not found: Error: Can't resolve '../../views/components/input/checkbox-5' in '/Users/conorpirie/projects/genny/alyson-v7/src/config/routes'
 @ ./config/routes/routes.web.js 1:153-203```

Reply:
Done.